libzhlgen/test/functions/mac/1.6.13/TextInput.zhl

```diff
@@ -0,0 +1,42 @@
+".554889e54157415653504189d74189f64889fb4889cee8":
```

Comment:
File starts with relative `.` signature, this continues from last position of last signature in previous file, might be unintended.